Shared/Samples/Control annotation sublayer visibility/ControlAnnotationSublayerVisibilityView.Model.swift

```diff
@@ -0,0 +1,126 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension ControlAnnotationSublayerVisibilityView {
+    /// The view model for this sample.
+    @MainActor
+    class Model: ObservableObject {
+        /// A map from the mobile map package.
+        @Published private(set) var map = Map()
+        
+        /// The mobile map package.
+        private var mobileMapPackage: MobileMapPackage!
+        
+        /// The annotation layer in the map.
+        private var annotationLayer: AnnotationLayer {
+            map.operationalLayers.first(where: { $0 is AnnotationLayer }) as! AnnotationLayer
```

Comment:
this isn't safe depending on when it is accessed. I would make it optional and set it once the map is loaded.